framework/xds_k8s_testcase.py

```diff
@@ -691,6 +691,84 @@ def assertAllBackendsReceivedRpcs(self, lb_stats):
                 msg=f"Backend {backend} did not receive a single RPC",
             )
 
+    def assertClientEventuallyReachesSteadyState(
+        self,
+        test_client: XdsTestClient,
+        rpc_type: str,
+        num_rpcs: int,
+        threshold_percent: int,
+        *,
+        retry_timeout: dt.timedelta = dt.timedelta(minutes=20),
+        retry_wait: dt.timedelta = dt.timedelta(seconds=2),
+        steady_state_delay: dt.timedelta = dt.timedelta(seconds=5),
+    ):
+        retryer = retryers.constant_retryer(
+            wait_fixed=retry_wait,
+            timeout=retry_timeout,
+            error_note=(
+                f"Timeout waiting for test client ${test_client.hostname} to"
```

Comment:
```suggestion
                f"Timeout waiting for test client {test_client.hostname} to"
```

lol, made this mistake myself a bunch of times